src/index.html

```diff
@@ -16,17 +16,46 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <div class="search-bar">
+      <div
+        class="search-bar__item search-bar__item--big"
+        data-qa="big"
+      >
+        <div class="icon-big"></div>
+        <form
+          action="#"
+          method="get"
+          class="form form--big"
+        >
+          <input
+            data-qa="keypress"
+            class="input input--big"
+            type="text"
+            name="city"
+            placeholder="Try  “Los Angeles“"
+          />
+        </form>
+      </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <div
+        class="search-bar__item search-bar__item--small"
+        data-qa="small"
+      >
+        <div class="icon-small"></div>
+        <form
+          action="#"
+          method="get"
+          class="form form--small"
+        >
+          <input
+            class="input input--small"
+            type="text"
+            name="city"
+            placeholder="Try  “Los Angeles“"
+          />
+        </form>
+      </div>
+    </div>
```

Comment:
Consider using semantic HTML tags. For example, you could use the `<section>` tag to wrap the `.search-bar` class. Semantic tags make your code more readable and accessible. They also help with SEO.